docs/src/components/MarkdownPage/AdditionalLinks.js

```diff
@@ -113,6 +116,22 @@ const AdditionalLinks = (props: Props) => {
           </BpkLink>
         )}
 
+        {/* Web sassdoc link */}
+        {platform && platform === PLATFORMS.web && sassdocId && (
+          <BpkLink
+            href={`/sassdoc/#${sassdocId}`}
+            blank
+            className={getClassName('bpkdocs-markdown-page__link')}
+          >
+            <img
+              width="55"
+              height="20"
+              src={`/${sassdocLogo}`}
+              alt={`View ${sassdocId} mixins and variables on our sassdoc`}
```

Comment:
```suggestion
              alt={`View ${sassdocId} mixins and variables on Sassdoc`}
```

Fixed capitalisation and removed first person usage to match the rest of the site